### ISSUE TITLE
Error message on Xash3d

### ISSUE THREAD

**Issue by ShamSmler** [sultim-t/xash-rt#128]
Getting all of these errors whenever I open up Xash3d. Have a feeling the issue is my Arc GPU
![Screenshot (9)](https://user-images.githubusercontent.com/104774415/223015505-49d56698-6566-4638-a5a0-e01d262f3a93.png)

**Comment by ShamSmler** [comment]
I did some more searching here and it seems other people with the same card are getting the same error and it's a known issue.

**Comment by Ozymandias81** [comment]
Happens the same for me, which I have and AMD Vishera FX Black Edition 3.5ghz 6 core cpu and a GeForce GTX 1060 Turbo 6gb
Running under Windows 10 Pro 64 bit
![image](https://user-images.githubusercontent.com/11136108/223778386-864a5f99-8fcd-4be6-8a6c-89f74c5c5399.png)

**Comment by apyoungblood** [comment]
I got the same error on my PC running a Geforce GTX 1060 6GB card as well. So I looked into it and there's some more details that other people may want to be aware of to make sense of why this doesn't work on our systems:

This mod - the xash3d - engine uses Vulkan for its renderer.

Nvidia chipsets officially started _hardware_ support for ray tracing in models from the 20-- series and newer. Typically they feature "RTX" in the title instead of "GTX." On the other hand if you have or want to purchase an AMD gpu - you will need a Radeon RX 6000 series or newer card.

Older Nvidia cards like the GTX 1060 do support ray tracing, but it's through a software feature patched into drivers from around mid 2019. That feature is called basic DirectX Raytracing (DXR). So if you've used it on your system - that's probably how. This also is incompatible with this particular mod, because DirectX is a different renderer than the Vulkan renderer used in this project.

So to recap if you're system gives you errors it's likely that some components don't fully support the feature. The easiest fix if you really want the feature would be to purchase and install a new GPU of the Nvidia 20-series RTX or greater, or AMD Radeon RX 6000 or greater. The newer hardware does ray tracing on the hardware itself, which is more ideal.

**Comment by Andrew-Cottrell** [comment]
FYI, apparently @yunyxn got it working on some GTX 10xx cards (e.g., GTX 1060 6 GB) with a modified [RayTracedGL1.zip](https://github.com/sultim-t/RayTracedGL1/files/10931937/RayTracedGL1.zip).

Ref: https://github.com/sultim-t/xash-rt/issues/2 and https://github.com/sultim-t/RayTracedGL1/pull/31
